src/Common/Core/Impl/Install/ISupportedRVersionList.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.Common.Core.Install {
+    public interface ISupportedRVersionList {
+        /// <summary>
+        /// Minimal supported R version major part such as 3 in 3.2
+        /// </summary>
+        int MinMajorVersion { get; }
+        /// <summary>
+        /// Minimal supported R version minor part such as 2 in 3.2
+        /// </summary>
+        int MinMinorVersion { get; }
```

Comment:
This is assuming that the third number in the version never matters (i.e. you can't specify at least 3.2.1, for example)  - are we sure that is the case?